src/App.js

```diff
@@ -1,26 +1,106 @@
 import "./App.css"
-import React, { useState } from "react"
-// import mockUsers from "./mockUsers.js"
+import React, { useState, useEffect } from "react"
+import mockUsers from "./mockUsers.js"
 import mockApartments from "./mockApartments.js"
 import { Routes, Route } from "react-router-dom"
 import Header from "./components/Header.js"
+import SignUp from "./pages/SignUp.js"
+import SignIn from "./pages/SignIn.js"
 import Home from "./pages/Home.js"
 import Index from "./pages/Index.js"
 import Show from "./pages/Show.js"
 import Footer from "./components/Footer.js"
 import NotFound from "./pages/NotFound.js"
 
 const App = () => {
-  // const [currentUser, setCurrentUser] = useState(mockUsers[0])
   const [apartments, setApartments] = useState(mockApartments)
+  const [currentUser, setCurrentUser] = useState(null)
+  console.log(currentUser)
+
+  // useEffect(() => {
+  //   const loggedInUser = localStorage.getItem("currentUser")
+  //   if (loggedInUser) {
+  //     setCurrentUser(JSON.parse(loggedInUser))
+  //   }
+  // }, [])
```

Comment:
The useEffect can be commented back in and you can remove the log for currentUser.